OpenRA.Mods.Common/Traits/GainsExperience.cs

```diff
@@ -123,6 +127,7 @@ public void ResolveOrder(Actor self, Order order)
 		}
 	}
 
+	// Remove this?
```

Comment:
Actually, the experience init is used (and if it isn't, we're either not at those missions yet or we're doing something wrong).